### DISCUSSION TITLE
Compatibility Mac OS

### DISCUSSION THREAD

**Discussion by proscrite** [PicoQuant/snAPI#7] [Q&A] [answered]
Is there any way to use snAPI on a Mac? It would be enough to use reading functions to open the .ptu files, I don't need to operate the device with a Mac

**Comment by tpoint75** [comment, marked as answer]
I think you have to use it on a virtual machine on Mac OS. I don't think to get time to compile snAPI for Mac or Arm64 anytime soon. Before that it would happen to make a Linux version.
